### PR TITLE
Added recipe for `torchtuples`

### PR DESCRIPTION
**Pull request by sarthakpati**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by sarthakpati** [comment]
@conda-forge/help-python this is ready for review!

File: recipes/docly/meta.yaml [hunk 1]
```diff
@@ -1,61 +0,0 @@
-{% set name = "docly" %}
```
Comment (dopplershift) [inline recipes/docly/meta.yaml:1]: Please don't modify any other recipes in here, just leave those files alone.

**Review by dopplershift — CHANGES_REQUESTED** [review]
Recipe looks good. Just need to undo the modification to the other recipe.

File: recipes/docly/meta.yaml (hunk 1 above)
Comment (sarthakpati) [reply recipes/docly/meta.yaml:1]: That was my bad. I had accidentally added that recipe earlier but reverted. This is that commit.

Comment (dopplershift) [reply recipes/docly/meta.yaml:1]: That's showing up as a change vs. master. If it was already merged, I'm not sure what to do other than rebase and eliminate your problematic commits.

**Comment by sarthakpati** [comment]
Closing in favor of #15473

**Pull request closed without merging**